### PR TITLE
Clarify update guidelines

### PR DESCRIPTION
**Pull request by BuonOmo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] ~Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?~
- [ ] ~Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?~
- [ ] ~Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?~
- [ ] ~Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?~

-----

Related to my work on #70863 and correction made by @carlocab (thanks)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 ### To submit a version upgrade for the `foo` formula
 
 * check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).
-* `brew bump-formula-pr --strict foo` with `--url=...` and `--sha256=...` or `--tag=...` and `--revision=...` arguments.
+* `brew bump-formula-pr --strict foo` with either:
+  * `--url=...` and `--sha256=...` 
+  * `--tag=...` and `--revision=...`
+  * `--version=...`
```
Comment (carlocab) [inline CONTRIBUTING.md:18]:
```suggestion
* `brew bump-formula-pr --strict foo` with one of:
  * `--url=...` and `--sha256=...`;
  * `--tag=...` and `--revision=...`; or,
  * `--version=...`
```
Thanks, @BuonOmo.

Comment (BuonOmo) [inline CONTRIBUTING.md:18]: If we need punctuations i still would not add the confusing `or`!
```suggestion
* `brew bump-formula-pr --strict foo` with one of:
  * `--url=...` and `--sha256=...`,
  * `--tag=...` and `--revision=...`,
  * `--version=...`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,7 +12,10 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 ### To submit a version upgrade for the `foo` formula
 
 * check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).
-* `brew bump-formula-pr --strict foo` with `--url=...` and `--sha256=...` or `--tag=...` and `--revision=...` arguments.
+* `brew bump-formula-pr --strict foo` with one of:
+  * `--url=...` and `--sha256=...`,
+  * `--tag=...` and `--revision=...`,
+  * `--version=...`.
```
Comment (carlocab) [inline CONTRIBUTING.md:18]:
```suggestion
* `brew bump-formula-pr --strict foo` with one of the following.
  * `--url=...` and `--sha256=...`
  * `--tag=...` and `--revision=...`
  * `--version=...`
```
If you don't want to use "or" we could just drop the punctuation in the list entirely.

Comment (BuonOmo) [reply CONTRIBUTING.md]: Perfect 👌

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,7 +12,10 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 ### To submit a version upgrade for the `foo` formula
 
 * check if the same upgrade has been already submitted by [searching the open pull requests for `foo`](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=is%3Apr+is%3Aopen+foo).
-* `brew bump-formula-pr --strict foo` with `--url=...` and `--sha256=...` or `--tag=...` and `--revision=...` arguments.
+* `brew bump-formula-pr --strict foo` with one of the following.
```
Comment (MikeMcQuaid) [inline CONTRIBUTING.md:15]:
```suggestion
* `brew bump-formula-pr --strict foo` with one of the following:
```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @BuonOmo!

**Pull request merged**